core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -227,6 +228,7 @@ object SparkEnv extends Logging {
       mockOutputCommitCoordinator: Option[OutputCommitCoordinator] = None): SparkEnv = {
 
     val isDriver = executorId == SparkContext.DRIVER_IDENTIFIER
+    val isContinuous = conf.getBoolean("spark.streaming.continuousMode", false)
```

Comment:
If I don't pass the right option at startup, I can never run a continuous processing query? I don't think that's the right model.